pkg/logging/logger/elasticsearch/es_logger.go

```diff
@@ -24,7 +24,7 @@ import (
 	"net/http"
 
 	"github.com/elastic/cloud-sdk-go/pkg/logging"
-	standardoutputlogger "github.com/elastic/cloud-sdk-go/pkg/logging/standardoutput"
+	logger "github.com/elastic/cloud-sdk-go/pkg/logging/logger/standardoutput"
 )
 
 const (
```

Comment:
@karencfv yeah both doesn't look very "friendly" :)
I renamed both the package and the structure so it can be referenced like this `elasticsearch.Logger` 

i hope it's better than what we had before